monkey/infection_monkey/telemetry/file_encryption_telem.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, entry: Tuple[str, str]):
 
         self._telemetry_entries.append(entry)
 
-    telem_category = TelemCategoryEnum.RANSOMWARE
+    telem_category = TelemCategoryEnum.FILE_ENCRYPTION
 
     def get_data(self):
-        return {"ransomware_attempts": self._telemetry_entries}
+        return {"files": self._telemetry_entries}
```

Comment:
Wouldn't something like `encryption_attempts` better explain this? The telem contains files and their encryption attempt results, not just files.

Reply:
I don't think so. I feel like `encryption_attempts` assumes failure, while `files` assumes success. `encryption_attempts` isn't really clear about what it contains. What does an "encryption attempt" involve? Timestamps? Scope? "file_encryption -> files" seems pretty clear to me: it contains information about the files that were encrypted.

Reply:
@shreyamalviya have agreed this is good enough for now, but will be revisited when we establish patterns and conventions for Action plugins (which don't exist right now).